streams/src/main/java/org/apache/kafka/streams/kstream/Windowed.java

```diff
@@ -26,26 +26,36 @@
  */
 public class Windowed<T> {
```

Comment:
@guozhangwang : This class explicitly talks about being "The windowed **key** interface used".  For consistency, should we then also replace `T` with `K` to highlight the focus on keys?  Alternatively, we could also keep `Windowed<T>` and simply remove the "key" focus from the docs -- it's only because we use `Windowed` for keys only in Streams, but there's nothing key-specific in this implementation.